auth/auth-router.js

```diff
@@ -1,11 +1,52 @@
 const router = require('express').Router();
+const bcrypt = require('bcryptjs')
+const jwt = require('jsonwebtoken')
 
-router.post('/register', (req, res) => {
+const Jokes = require('../jokes/jokes-model')
```

Comment:
not a big issue but I noticed your joke's model is capitalized but should be lower case